### PR TITLE
Use automatic broadcasting in distributions

### PR DESCRIPTION
**Pull request by toslunar**

### CODE REVIEW

File: chainer/distributions/bernoulli.py [hunk 1]
```diff
@@ -123,8 +122,8 @@ def prob(self, x):
             valid = cuda.cupy.bitwise_or(x.array == 0, x.array == 1)
         else:
             valid = numpy.bitwise_or(x.array == 0, x.array == 1)
-        ret = x * broadcast.broadcast_to(self.p, x.shape) \
-            + (1 - x) * (1 - broadcast.broadcast_to(self.p, x.shape))
+        ret = x * self.p \
+            + (1 - x) * (1 - self.p)
```
Comment (YoshikawaMasashi) [inline chainer/distributions/bernoulli.py]: Is it better to write with one line?

File: chainer/distributions/normal.py [hunk 2]
```diff
@@ -76,9 +75,9 @@ def batch_shape(self):
         return self.loc.shape
 
     def cdf(self, x):
-        return 0.5 * erfc.erfc(-(
-            x - broadcast.broadcast_to(self.loc, x.shape))
-            / (2 ** 0.5 * broadcast.broadcast_to(self.scale, x.shape)))
+        return 0.5 * erfc.erfc(
+            (self.loc - x)
+            / (2 ** 0.5 * self.scale))
```
Comment (YoshikawaMasashi) [inline chainer/distributions/normal.py]: ditto (Is it better to write with less line?

File: chainer/distributions/normal.py [hunk 3]
```diff
@@ -142,8 +138,8 @@ def support(self):
 
     def survival_function(self, x):
         return 0.5 * erfc.erfc(
-            (x - broadcast.broadcast_to(self.loc, x.shape))
-            / (2 ** 0.5 * broadcast.broadcast_to(self.scale, x.shape)))
+            (x - self.loc)
+            / (2 ** 0.5 * self.scale))
```
Comment (YoshikawaMasashi) [inline chainer/distributions/normal.py]: ditto (Is it better to write with less line?

**Comment by toslunar** [comment]
Fixed.

**Comment by YoshikawaMasashi** [comment]
jenkins, test this please.

**Pull request merged**